### PR TITLE
ObjectRecognizerTest and GenderRecognizerTest added

### PR DESCRIPTION
**Pull request by Joselian2204**

### CODE REVIEW

**Review by GDayne — COMMENTED** [review]
Test working

File: tests/gender_recognizer_test.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import unittest
+
+from models.Recognizer.GenderRecognizer import GenderRecognizer
+from models.Recognizer.DetectedFrame import DetectedFrame
+
+class TestGenderRecognizer(unittest.TestCase):
+
+    def setUp(self):
+        self.recognizer = GenderRecognizer()
+        # Rutas de imágenes de prueba para mujeres y hombres (asegúrate de tener estas imágenes)
```
Comment (Eynar92) [inline tests/gender_recognizer_test.py:10]: TODO: We must translate our comments

**Review by Eynar92 — COMMENTED** [review]
We must translate our comments

File: tests/gender_recognizer_test.py (hunk 1 above)
Comment (Joselian2204) [reply tests/gender_recognizer_test.py:10]: Done on GenderRecognizer

**Pull request merged**